docker-compose.yml

```diff
@@ -1,7 +1,37 @@
 version: '2'
 services:
+  app:
+    build: .
+    image: keboola/ex-ftp
+    command: composer ci
+    links:
+      - ftp
   dev:
     build: .
     volumes:
       - ./:/code
       - ./data:/data
+    links:
+      - ftp
+  ftp:
+    image: stilliard/pure-ftpd
+    environment:
+      FTP_USER_NAME: ftpuser
+      FTP_USER_PASS: userpass
+      FTP_USER_HOME: /home/ftpusers/
+      PUBLICHOST: ftp
+    volumes:
+      - ./tests/ftpInitContent:/home/ftpusers
+  tests:
+      build:
+        context: .
+        dockerfile: Dockerfile-tests
+      volumes:
+        - ./:/code
+        - ./data:/data
+      environment:
+        - XDEBUG_CONFIG=remote_enable=1 remote_mode=req remote_port=9000 remote_host=172.20.0.1 remote_connect_back=0
```

Comment:
Ta IP adresa GW se bude asi u každého měnit. Zkusím do budoucna přijít na jak jí propašovat dynamicky.

Reply:
host.docker.internal

Reply:
(nefunguje ale na linuxu, bohužel)

Reply:
Já mám linux. Počkám jestli to dodělají a pak to opravím.